plugins/env/app/models/environment_variable.rb

```diff
@@ -48,6 +49,22 @@ def serialize(variables, env_scopes)
       end.join("\n")
     end
 
+    def env_vars_from_db(project, deploy_group)
+      variables = nested_variables(project)
+      variables.sort_by!(&:priority)
+      variables.each_with_object({}) do |ev, all|
+        all[ev.name] = ev.value if !all[ev.name] && ev.matches_scope?(deploy_group)
+      end
+    end
+
+    def env_vars_from_repo(env_repo_name, project, deploy_group)
+      path = "generated/#{project.permalink}/#{deploy_group.permalink}.env"
+      content = GITHUB.contents(env_repo_name, path: path, headers: {Accept: 'applications/vnd.github.v3.raw'})
+      Dotenv::Parser.call(content)
+    rescue StandardError => e
```

Comment:
Ruby by default rescues `StandardError` so this is a little redundant. You could do something like
```ruby
rescue
   raise Samson::Hooks::UserError, "Cannot download env file #{path} from #{env_repo_name} (#{$!.message})"
end
```
`$!` is a ruby global variable which points to the last exception raised.

Reply:
?

Reply:
I think rubocop wants StandardError ... also `=> e` is simpler to understand ... (either way is fine with me 🤷‍♂️ )